test/turbopack-tests-manifest.json

```diff
@@ -3308,21 +3310,21 @@
       "app dir - metadata dynamic routes icon image routes should render apple icon with dynamic routes",
       "app dir - metadata dynamic routes icon image routes should render icon with dynamic routes",
       "app dir - metadata dynamic routes should error when id is missing in generateImageMetadata",
-      "app dir - metadata dynamic routes should inject dynamic metadata properly to head",
-      "app dir - metadata dynamic routes should pick configured metadataBase instead of deployment url for canonical url",
-      "app dir - metadata dynamic routes should use localhost for local prod and fallback to deployment url when metadataBase is falsy",
-      "app dir - metadata dynamic routes social image routes should fill params into dynamic routes url of metadata images",
       "app dir - metadata dynamic routes social image routes should handle manifest.[ext] dynamic routes",
       "app dir - metadata dynamic routes social image routes should render og image with opengraph-image dynamic routes",
-      "app dir - metadata dynamic routes social image routes should render og image with twitter-image dynamic routes",
-      "app dir - metadata dynamic routes social image routes should support generate multi images with generateImageMetadata",
       "app dir - metadata dynamic routes text routes should handle robots.[ext] dynamic routes"
     ],
     "failed": [
       "app dir - metadata dynamic routes should error when id is missing in generateSitemaps",
       "app dir - metadata dynamic routes should generate unique path for image routes under group routes",
+      "app dir - metadata dynamic routes should inject dynamic metadata properly to head",
```

Comment:
https://github.com/vercel/next.js/pull/57431 is in canary, so rebasing should make pass some of these tests

Reply:
It's updated from the daily test run... The next run will make it pass.

Reply:
yeah but there's passing test and don't need to move to failed manually? but it's indifferent